build/gen_stub.php

```diff
@@ -5360,8 +5725,10 @@ function(?ArgInfo $aliasArg, ?ArgInfo $aliasedArg) use ($aliasFunc, $aliasedFunc
         }
 
         foreach ($methodSynopses as $filename => $content) {
-            if (file_put_contents("$methodSynopsesDirectory/$filename", $content)) {
-                echo "Saved $filename\n";
+            if (!file_exists("$methodSynopsesDirectory/$filename")) {
```

Comment:
Maybe in order to avoid ambiguity, a debug message should be added like `echo "$filename already exists\n";`

Reply:
I don't think this is a good idea. Considering my end goal is to be able to just run ``../php-src/build/gen_stub.php --generate-methodsynopses ../PHP-8.3/ ./../docs-php/en/`` and it add the missing file, it would produce a shit ton of output as most of the files would already exist